app/stores/AppStore.js

```diff
@@ -11,8 +11,13 @@ export default class AppStore extends Store {
     this.actions.goToRoute.listen(this._updateRouteLocation);
   }
 
+  @computed get currentRoute() {
```

Comment:
This looked ugly to me: `return router.location ? router.location.pathname === screenRoute : false;`
But your point is valid. I leave the decision to you. I will change it back if you want.